overlord/snapstate/snapmgr.go

```diff
@@ -634,7 +634,7 @@ func (snapst *SnapState) RefreshInhibitProceedTime(st *state.State) time.Time {
 	}
 	// TODO: state is needed for when configurable max inhibition
 	// is introduced (i.e. "core.refresh.max-inhibition-days").
```

Comment:
I think we can safely remove this TODO now?

Reply:
I presume...